go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/prometheus/prometheus
 
-go 1.20
+go 1.21.2
```

Comment:
We don't want this.  Prometheus intends to be compilable by the last two versions of Go.

Reply:
Ok but then how is the version of go supposed to be upgraded

Reply:
First the build image is updated, e.g.: https://github.com/prometheus/golang-builder/pull/227
Then when we next do a release of Prometheus it will use that version by default.

Reply:
Ohk Got it